gateway-contracts/contracts/interfaces/IGatewayConfig.sol

```diff
@@ -108,6 +108,16 @@ interface IGatewayConfig {
     /// @param nKmsNodes The number of KMS nodes.
     error InvalidHighUserDecryptionThreshold(uint256 userDecryptionThreshold, uint256 nKmsNodes);
 
+    /**
+     * @notice Emitted when all the pausable gateway contracts are paused.
+     */
+    event AllGatewayContractsPaused();
```

Comment:
nit: we decided long ago not to use past tense for the events, so maybe we should rename these events to: `PauseAllGatewayContracts` and `UnpauseAllGatewayContracts`